web_src/js/features/repo-findfile.test.js

```diff
@@ -0,0 +1,35 @@
+import {strSubMatch, calcMatchedWeight, filterRepoFilesWeighted} from './repo-findfile.js';
+import {expect} from '@playwright/test';
```

Comment:
```suggestion
```

We should use the jest global instead. I think playwright `expect` is mean to only be used in playwright tests.

Reply:
Hmm, by mistake.